fvm/src/lib.rs

```diff
@@ -1,10 +1,13 @@
+#![feature(int_roundings)]
```

Comment:
We shouldn't use nightly features.

Reply:
Should I just manually do the rounding for floor/ceil then? Or is there some other builtin way to specify round up/down when dividing?

Reply:
Hm. That's annoying. Yeah, I think we should do this manually. It looks like the implementation is a bit unstable and will actually be changing in the near future.